libs/community/langchain_community/utilities/sql_database.py

```diff
@@ -410,30 +421,45 @@ def _execute(
                 elif self.dialect == "postgresql":  # postgresql
                     connection.exec_driver_sql("SET search_path TO %s", (self._schema,))
 
-            cursor = connection.execute(text(command))
+            if isinstance(command, str):
+                command = text(command)
+            elif isinstance(command, Executable):
+                pass
+            else:
+                raise TypeError(f"Query expression has unknown type: {type(command)}")
```

Comment:
That's the third feature of this patch. Same here: Should this capability be added to the documentation?

Reply:
Dito. Fixed with cf57a092483b.